ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_paddle_math.py

```diff
@@ -1027,20 +1026,21 @@ def test_paddle_cumprod(
         on_device=on_device,
         x=x[0],
         dim=axis,
-        rtol=1e-01,
-        atol=1e-01,
     )
 
 
 # gcd
 @handle_frontend_test(
     fn_tree="paddle.gcd",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("valid"),
-        num_arrays=2,
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_value=-100,
+        max_value=100,
         min_num_dims=1,
-        safety_factor_scale="log",
-        large_abs_safety_factor=2,
+        max_num_dims=3,
```

Comment:
Not sure if we want to restrict the `max_num_dims` to `3` only since the native function supports ND inputs